### ISSUE TITLE
[RFE] openshift-monitoring namespace is not collected via containerCPU/Memory indexing.

### ISSUE THREAD

**Issue by afcollins** [cloud-bulldozer/kube-burner#546]
**Is your feature request related to a problem? Please describe.**
The conatinerCPU/Memory indexed metrics that show for individual nodes do not index openshift-monitoring namespace.

**Describe the solution you'd like**
Add "monitoring" to any query that filters out specific openshift namespaces.

**Describe alternatives you've considered**
Remove explicitly declared openshift-* namespaces and simply index all openshift-* namespaces so we can monitor newly-introduced ns's without adding each new one specifically.

**Additional context**
None. Fin.

**Comment by rsevilla87** [comment]
> **Is your feature request related to a problem? Please describe.** The conatinerCPU/Memory indexed metrics that show for individual nodes do not index openshift-monitoring namespace.
> 
> **Describe the solution you'd like** Add "monitoring" to any query that filters out specific openshift namespaces.
> 
> **Describe alternatives you've considered** Remove explicitly declared openshift-* namespaces and simply index all openshift-* namespaces so we can monitor newly-introduced ns's without adding each new one specifically.
> 
> **Additional context** None. Fin.

makes sense to scrape openshift-* namespaces, we'd have to filter out some not interesing pods from daemonsets though
